release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -1930,6 +1935,45 @@ With this release, the CCO no longer checks for the nonexistent permission.
 
 * Previously, proxy information that was set in the `install-config.yaml` file was not applied to the bootstrap process. With this release, proxy information is applied to bootstrap ignition data, which is then applied to the bootstrap machine. (link:https://issues.redhat.com/browse/OCPBUGS-12890[*OCPBUGS-12890*])
 
+* Previously, the {ibm-power-server-name} platform had no Dynamic Host Configuration Protocol (DHCP) network name, the DHCP resource was not deleted. With this release, a check looks for any DHCP resources with an `ERROR` state and deletes them so that this issue no longer occurs. (link:https://issues.redhat.com/browse/OCPBUGS-35224[*OCPBUGS-35224*])
```

Comment:
Seems to be missing a word, maybe:
```suggestion
* Previously, when the {ibm-power-server-name} platform had no Dynamic Host Configuration Protocol (DHCP) network name, the DHCP resource was not deleted. With this release, a check looks for any DHCP resources with an `ERROR` state and deletes them so that this issue no longer occurs. (link:https://issues.redhat.com/browse/OCPBUGS-35224[*OCPBUGS-35224*])
```

Reply:
:heavy_check_mark:

Reply:
:heavy_check_mark: